src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import './variables';
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    background-image: url('src/images/star-empty.png');
```

Comment:
Verify that the image `src/images/star-empty.png` exists at the specified path. Incorrect paths will result in missing images.